skrub/_gap_encoder.py

```diff
@@ -651,6 +651,36 @@ class GapEncoder(BaseEstimator, TransformerMixin):
     fitted_models_: list[GapEncoderColumn]
     column_names_: list[str]
 
+    @classmethod
+    def _merge(cls, transformers_list):
```

Comment:
What's the type of `transformers_list`? Here's my guess, please correct it if necessary:
```suggestion
    def _merge(cls, transformers_list: list[GapEncoder]):
```